code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -48,7 +49,8 @@ var/list/gaslist_cache = null
 	src.volume = volume
 	last_share = 0
 	fuel_burnt = 0
-
+	create_reagents(1000)
+	reagents.set_reacting(FALSE) // no

```

Comment:
This is the kind of thing that might have to be lazyinit'ed also, it's a datum not a list so we don't have the macros for that, but every gas mixture doesn't need a reagent datum, only ones with reagents